cost-analyzer/values.yaml

```diff
@@ -217,8 +217,19 @@ global:
     # iam.amazonaws.com/role: role-arn
   additionalLabels: {}
 
-  containerSecuritycontext: {}
-    # readOnlyRootFilesystem: true
+  securityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
+    fsGroup: 65534
+    runAsGroup: 65534
+    runAsUser: 65534
+  containerSecurityContext:
+    allowPrivilegeEscalation: false
```

Comment:
After looking at this more carefully, you've already got `allowPrivilegeEscalation`. It'd probably be easiest for you to just add a second line `privileged: false` beneath it.

Reply:
for sure, I was thinking it was more complex. thanks!